qiskit_experiments/database_service/db_analysis_result.py

```diff
@@ -526,3 +526,19 @@ def __repr__(self):
             out += f", {key}={repr(val)}"
         out += ")"
         return out
+
+    def __json_encode__(self):
+        return {
+            "name": self._name,
+            "value": self._value,
+            "device_components": self._device_components,
+            "experiment_id": self._experiment_id,
+            "result_id": self._id,
+            "chisq": self._chisq,
+            "quality": self._quality,
+            "extra": self._extra,
+            "verified": self._quality_verified,
+            "tags": self._tags,
+            "service": self._service,
```

Comment:
How can this be json serialized?

Reply:
It probably doesn't deserialize correctly (like with Backends) due to account management